dotCMS/src/main/java/com/dotcms/http/server/mock/MockHttpServer.java

```diff
@@ -75,6 +85,53 @@ public void start() {
         }
     }
 
+    private static String getErrorMessage(List<MockHttpServerContext> contexts,
+            RequestContext requestContext) {
+        return "New Request:" + contexts.stream()
+                    .flatMap(context -> context.getRequestConditions().stream()
```

Comment:
If intend to use `flatMap()` then I think the code you will need it's actually:
```
contexts.stream()
                    .flatMap(context -> context.getRequestConditions().stream())
                    .map(reqCondition -> requestCondition.getMessage(requestContext)
```

Reply:
done https://github.com/dotCMS/core/pull/24625/commits/989229988f13072ff026d0e585bd2ddb61304cc3#diff-bb8e6d4042484875cf32b80f746a82828bf3c93ebaa404c732329defc4f9b003R91